lib/raster3d/test/test_put_get_value_large_file.c

```diff
@@ -130,6 +130,10 @@ int test_large_file(int depths, int rows, int cols, int tile_size)
             for(x = 0; x < region.cols; x++) {
                 /* Check the counter as cell value */
                 Rast3d_get_value(map, x, y, z, &value, DCELL_TYPE);
+                /* UNCERTAIN SOLUTION
+                warning: using floating point absolute value
+                function 'fabs' when argument is of integer type 
+                -- either rearrange parenthesis or use abs() */
                 if(fabs(value - (double)(count) > EPSILON)) {
```

Comment:
Please, see comment in code, what is the correct solution?

Reply:
I didn't examine the broader context, but given that value is DCELL, that count is casted to double, the statement itself, the fact that `>` poperator returns 0 or 1, and the following message, it looks like the intention was a classic `|a - b| > threshold` where `|x|` signifies absolute value of `x`. Conclusion: the parenthesis is wrong. It is a test, so perhaps the problem it never surfaced.

Reply:
Following the logic of the code, the correct solution must be `if (fabs(value - (double)(count)) > EPSILON) {`

Reply:
Again, figured `|a - b| > threshold` must be correct. Thanks for confirmation.

Reply:
updated as suggested